.travis.yml

```diff
@@ -1,15 +1,16 @@
-language: node_js
-node_js:
-    - "0.10"
+language: php
+php:
+    - "5.6"
```

Comment:
(5.7 ? I guess you mean 7 :))

Well, actually before this PR, we've been testing only with PHP 5.5 (provided by `ppa:ondrej/php5`). Given that we have 2 code bases to test, to not run the JS tests 3 times, we either have to choose a PHP version or I have to change the JS test be executed only when a specific PHP version is configured but I find that a bit weird to be honest.